styles/docs.css

```diff
@@ -180,10 +180,10 @@ nav {
     border-right: 1px solid #f0f0f0;
     display: -webkit-flex;
     display: -ms-flexbox;
-    display: flex;
+    display:         flex;
```

Comment:
As I said before, CSSComb does not cope well with these multiple displays. You suggested switching it out for something else? We only need the `-webkit-` for Android really.